File: help/pt_pt/exercicios_da_ficha_3/Ficha03_019/Ficha03_019.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Ângulo (rad) \t: " );// print text
    ang = parseFloat(prompt("Ângulo (rad) \t: " ,"0.0") );
    
    document.write(ang + "\n");// print input 
    document.write("Seno \t\t= " + Math.sin( ang ) + "\n" );
    document.write("Coseno \t\t= " + Math.cos( ang ) + "\n" );
    document.write("Tangente \t= " + Math.tan( ang ) );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:32

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_019/Ficha03_019.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha03_019 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Ângulo (rad) \t: " );
        double ang = Double.valueOf(keyb.nextLine());
        System.out.print("Seno \t\t= " + Math.sin( ang ) + "\n" );
        System.out.print("Coseno \t\t= " + Math.cos( ang ) + "\n" );
        System.out.print("Tangente \t= " + Math.tan( ang ) );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:32
